styles/Home.module.css

```diff
@@ -6,11 +6,6 @@
   min-height: 100vh;
```

Comment:
Couldn't this file be deleted and instead any required styling is moved to `globals.css`?

Reply:
yes, we can do that, it's just the way it was initialized by next js; change it? yes/no

Reply:
Yes, please remove files we are not actively using, as it requires maintenance and can cause side effects